_layouts/default.html

```diff
@@ -11,7 +11,7 @@
     </head>
     <body>
         <header>
-            <h1><a href="/"><img src="{{ root }}wobscale.svg" alt="Wobscale"></a></h1>
+            <h1><a href="{{ root }}"><img src="{{ root }}wobscale.svg" alt="Wobscale"></a></h1>
```

Comment:
I think this one wants to be root/index.html because otherwise other pages (e.g. the CoC one) don't link back to the index